### PR TITLE
Add a team  label to enable cross-namespace communication

### PR DESCRIPTION
**Pull request by dimberman**

### CODE REVIEW

File: astronomer_namespace.tf [hunk 1]
```diff
@@ -3,6 +3,7 @@ resource "kubernetes_namespace" "astronomer" {
     name = var.astronomer_namespace
     labels = {
       istio-injection = "enabled"
+      team: "operations"
```
Comment (schnie) [inline astronomer_namespace.tf]: Think this `:` needs to be `=`

Comment (dimberman) [reply astronomer_namespace.tf]: thanks @schnie fixed

**Comment by schnie** [comment]
@dimberman so if someone wants to use this that doesn't have the ability to use our terraform, this would be an additional install step right?

File: astronomer_namespace.tf (hunk 1 above)
Comment (jpweber) [reply astronomer_namespace.tf]: I this is nitpicky, but I feel like "team" as label key doesn't quite fit. I know its sort of redundant but you could have a label that is simply `name: astronomer` which is really what we're trying to do is identify the astronomer namespace, but `NetworkPolicies` work on labels.

Comment (schnie) [reply astronomer_namespace.tf]: Yea, I like that too - or maybe `platform`? Either one

Comment (dimberman) [reply astronomer_namespace.tf]: I've set the namespace label to "name: astronomer"

**Comment by dimberman** [comment]
@schnie if somebody wants to have a multi-namespace astronomer cluster and can not use our terraform then yes, they would need to add this label.

**Comment by schnie** [comment]
I'm hoping we can avoid this PR. If we can configure KEDA like we do Prometheus in the pgbouncer `NetworkPolicy` we should be able to get around this right?

I'm just trying to keep any platform level logic from bleeding up into our terraform, or adding install steps for folks not using terraform -- if possible.

**Comment by sjmiller609** [comment]
> I'm hoping we can avoid this PR. If we can configure KEDA like we do Prometheus in the pgbouncer `NetworkPolicy` we should be able to get around this right?
> 
> I'm just trying to keep any platform level logic from bleeding up into our terraform, or adding install steps for folks not using terraform -- if possible.

yes this is possible by using namespaceSelector: {} (i.e. 'ANY') with AND on pod selector. I agree this is the best option for this case

**Pull request merged**